File: Looping/Loop inside loop/LoopInLoop.java
public class LoopInLoop {
	public static void main(String[] args) throws Exception {
		System.out.print("\n");
		for (int vert = 1; vert <= 3; vert++) {
			// Nested Loops gave me anxiety...
			for (int horz = 0; horz < 8; horz++) {
				System.out.print(vert + " ");
			}
			System.out.print("\n");
		}
	}
}

File: Looping/Loop inside loop/Other Lang/C/LoopInLoop.c
#include <stdio.h>

void main() {
	for (int vert = 1; vert <= 3; vert++) {
		//Nested Loops Gave Me Anxiety...
		for (int horz = 0; horz < 8; horz++) {
			printf("%d ", vert);
		}
		printf("\n");
	}
}
